README.md

```diff
@@ -20,15 +20,25 @@ forced timeouts. FTP Secure (FTPS) is also supported.
 
     # A more concrete example:
 
-    # Mixin the +NiFTP+ module, which provides the +ftp+ method.
+    # Mixin the +NiFTP+ module, which provides the +ftps+ method.
```

Comment:
This needs to be `ftp` still, as `ftp` is still the name of the method we call in this gem.